src/components/component.ts

```diff
@@ -146,11 +194,11 @@ export class Component {
     });
     this._cssClasses = new Utils.Set<string>();
 
-    this._backgroundContainer = this._element.append("g").classed("background-container", true);
+    this._backgroundContainer = this._element.append("svg").classed("background-container", true);
```

Comment:
Eventually I'd like to move these away from Component and have a separate thing own SVG specific concepts (either a mixin, or introduce a new idea of a RenderTarget).

If you're worried about unused DOM nodes, we can lazily instantiate/add these when they're used.